tests/test_ops.py

```diff
@@ -223,7 +239,7 @@ def test_create_ops_arg_function(self, read):
         u = OpsAccessible('u', dtype=np.float32, read_only=read)
         dat = OpsDat('u_dat')
         stencil = OpsStencil('stencil')
-        info = AccessibleInfo(u, None, None)
+        info = AccessibleInfo(u, None, None, None)
```

Comment:
@btrb for later PR... namedtuple can be engineering to use default arguments. Could be handy since many times most arguments are just None

Reply:
oki